components/NavigationDrawer/examples/BottomDrawerWithHeaderExample.swift

```diff
@@ -75,6 +80,36 @@ class BottomDrawerWithHeaderExample: UIViewController, MDCBottomDrawerViewContro
     present(bottomDrawerViewController, animated: true, completion: nil)
   }
 
+  private func layoutContentViewController() {
+    let button = MDCButton(frame: .zero)
+    button.setTitle("Expand", for: .normal)
+    button.sizeToFit()
+    button.addTarget(self, action: #selector(expandBottomDrawer), for: .touchUpInside)
+    button.center = CGPoint(x: view.frame.width / 2, y: 50)
+    button.backgroundColor = colorScheme.primaryColor
+    contentViewController.view.addSubview(button)
+  }
+
+  @objc func expandBottomDrawer() {
+    var height: CGFloat = 0
+    if (!fullScreen) {
+      height = 100
+    } else {
+      height = 2000
+    }
+    bottomDrawerViewController.animate(toPreferredContentHeight: height, withDuration: 5) { _ in
```

Comment:
can we please make this a lower duration that makes sense for the transition, maybe 0.2?